src/geometry/point.rs

```diff
@@ -458,16 +458,8 @@ where
     DefaultAllocator: Allocator<T, D>,
 {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{{")?;
-
-        let mut it = self.coords.iter();
-
-        write!(f, "{}", *it.next().unwrap())?;
-
-        for comp in it {
-            write!(f, ", {}", *comp)?;
-        }
-
+        write!(f, "Point {{")?;
```

Comment:
I'd rather omit this wrapping for concision, personally. It's also unconventional for `Display` impls in general.